src/CookieStore.ts

```diff
@@ -101,13 +101,20 @@ class CookieStore {
     const persistedCookies = localStorage.getItem(PERSISTENCY_KEY)
 
     if (persistedCookies) {
-      const parsedCookies: [string, [string, any]] = JSON.parse(
-        persistedCookies
-      )
-
-      parsedCookies.forEach(([origin, cookie]) => {
-        this.store.set(origin, new Map(cookie))
-      })
+      try{
```

Comment:
I've pushed the basic Prettier config to the `master`. Could you please update your feature branch and format this file? Thanks.